tests/tasks/test_http.py

```diff
@@ -126,3 +127,30 @@ def test_http_get_define():
     ):
         http = Http(name, url)
         assert http.task_params == expect_task_params
+
+
+def test_http_params_conversion():
```

Comment:
due to you already copy and paste it from file `test_http_conversion.py` I thing you can remove the file `test_http_conversion.py`.

Reply:
BTW we should add one more test for the warning when user still pass list to http task parameter `http_params`. we have an example in https://github.com/apache/dolphinscheduler-sdk-python/blob/f82e07d232380fe9f5582db4dcca920ae00befd4/tests/tasks/test_sub_workflow.py#L110-L136 wish it can help you